contracts/libraries/TwabLib.sol

```diff
@@ -89,10 +89,16 @@ library TwabLib {
         accountDetails.balance = (_accountDetails.balance - _amount).toUint208();
     }
 
-    /// @notice Calculates the average balance held by a user for a given time frame.
-    /// @param _startTime The start time of the time frame.
-    /// @param _endTime The end time of the time frame.
-    /// @return The average balance that the user held during the time frame.
+    /** @notice Calculates the average balance held by a user for a given time frame.
+      * @dev    Finds the average balance between start (1609506061) and end (1639311132) timestamp epochs.
```

Comment:
Why give specific timestamps?  I find that confusing.  but otherwise this looks good

Reply:
LoL... good point. No timestamp examples needed. Fixed.